### DISCUSSION TITLE
Convert Github User to Organization (keeping the same name)

### DISCUSSION THREAD

**Discussion by TechDufus** [community/community#23071] [New to GitHub]
<p>I had created a 2nd GitHub user account under a username that I now want to exist as an organization instead.</p>
<p>I have deleted the entire GitHub account, which I thought would free up this username to allow me to create a new (free) organization with the same username, but I’m being told that the username is unavailable.</p>
<p>Do I simply need to wait longer?</p>

**Comment by jsoref** [comment, marked as answer]
<p>For reference, this is what you were supposed to do:<br>
<a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/converting-a-user-into-an-organization" class="onebox" target="_blank" rel="noopener">https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/converting-a-user-into-an-organization</a></p>
<p>I’d contact:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2119212/181074684-4275b2f8-24db-499f-a45f-d6e6b7d8b4b2.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com/" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com/" target="_blank" rel="noopener">GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<p>
(but, be patient)</p>
